executor/prepared_test.go

```diff
@@ -301,7 +301,11 @@ func (s *testSerialSuite) TestPlanCacheClusterIndex(c *C) {
 func (s *testPrepareSuite) TestPlanCacheWithDifferentVariableTypes(c *C) {
 	store, dom, err := newStoreWithBootstrap()
 	c.Assert(err, IsNil)
-	tk := testkit.NewTestKit(c, store)
+	se, seErr := session.CreateSession(store)
+	c.Assert(seErr, IsNil)
+	// disable GetVar Fold
+	se.GetSessionVars().EnableGetVarFold = false
```

Comment:
Why we need the session variable `EnableGetVarFold`? Is there any performance fallback after folding `GetVar`?

Reply:
In order to pass this test, I use this `EnableGetVarFold` to disable the fold logic.
After read your message in `common_plans.go`,  I notice its incorrect.
I will remove this variable.